lib/Backend/Func.h

```diff
@@ -52,6 +52,36 @@ struct Cloner
     bool clonedInstrGetOrigArgSlotSym;
 };
 
+/*
+* This class keeps track of various information required for Stack Arguments optimization with formals.
+*/
+class StackArgWithFormalsTracker
+{
+private:
+    BVSparse<JitArenaAllocator> *  formalsArraySyms;    //Tracks Formal parameter Array - Is this Bv required explicitly?
+    StackSym**                     formalsIndexToStackSymMap; //Tracks the stack sym for each formal
+    StackSym*                      m_scopeObjSym;   // Track the stack sym for the scope object that is created.
```

Comment:
Done.

---
In reply to: [68144068](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r68144068) [](ancestors = 68144068)